profileViewer/modelViewer.js

```diff
@@ -137,6 +151,9 @@ function animationFrameCallback() {
     Object.values(activeModel.motionController.components).forEach((component) => {
       const componentNodes = activeModel.nodes[component.id];
 
+      if (!componentNodes)
```

Comment:
Add a comment indicating that this can be skipped because errors are notified at load time?